allcontainers.js

```diff
@@ -98,7 +105,17 @@ function allContainers (opts) {
         return result.emit('error', err)
       }
 
-      containers.forEach(emit)
+      var activeContainers = [];
+      function findPausedContainers(container) {
+        if (container.Status.indexOf("Paused") !== -1) {
+          pausedPreheatContainers[container.Id] = container;
+        } else {
+          activeContainers.push(container);
+        }
+      }
```

Comment:
`reduce` with side effects seems like somewhat of an abuse, and looping twice somewhat wasteful, which is why I did it this way. I'm happy to do whatever you want though.